### PR TITLE
[HER-14] Setup Apollo client

### PR DESCRIPTION
**Pull request by Urzica-Platon-Casian**
## Description
Setup Apollo Client for GraphQL integration.

## Related
[HER-14](https://nerds-interns.atlassian.net/jira/software/projects/HER/boards/1?issueParent=10006&selectedIssue=HER-14)

### CODE REVIEW

File: src/index.jsx [hunk 1]
```diff
@@ -1,8 +1,29 @@
...
 import ReactDOM from 'react-dom'
+import {
+  ApolloClient, gql,
+  InMemoryCache
+} from '@apollo/client'
 
 import { App } from 'app'
 
+const client = new ApolloClient({
+  uri: 'https://48p1r2roz4.sse.codesandbox.io',
```
Comment (miutamihai) [inline src/index.jsx:11]: Is this really our api url? :smile:

**Review by miutamihai — CHANGES_REQUESTED** [review]

File: src/index.jsx [hunk 2]
```diff
@@ -1,8 +1,29 @@
...
+  cache: new InMemoryCache(),
+})
+
+client
+  .query({
```
[the remark this answers is not in the document]
Comment (Urzica-Platon-Casian) [reply src/index.jsx]: Done

File: src/index.jsx (hunk 1 above)
Comment (Urzica-Platon-Casian) [reply src/index.jsx]: Done

File: src/index.jsx [hunk 3]
```diff
@@ -1,29 +1,17 @@
...
-  ApolloClient, gql,
+  ApolloClient,
   InMemoryCache
 } from '@apollo/client'
 
 import { App } from 'app'
 
-const client = new ApolloClient({
-  uri: 'https://48p1r2roz4.sse.codesandbox.io',
+ApolloClient({
```
Comment (miutamihai) [inline src/index.jsx:10]: We DO need the client in a constant, so that we can export it and use it throughout the application.

Comment (miutamihai) [reply src/index.jsx:10]: Also, it should be located in a separate file under the `/startup` directory.

**Review by miutamihai — CHANGES_REQUESTED** [review]

File: src/index.jsx (hunk 3 above)
Comment (Urzica-Platon-Casian) [reply src/index.jsx]: Done

**Review by miutamihai — APPROVED** [review]

**Review by mihairadulescu — APPROVED** [review]

**Pull request merged**